example/src/main/java/io/github/btkelly/gandalf/example/activities/SplashActivity.java

```diff
@@ -15,8 +15,85 @@
  */
 package io.github.btkelly.gandalf.example.activities;
 
+import android.annotation.SuppressLint;
+import android.app.AlarmManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.preference.PreferenceManager;
+import android.support.annotation.RawRes;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import io.github.btkelly.gandalf.activities.GandalfActivity;
+import io.github.btkelly.gandalf.example.R;
+import io.github.btkelly.gandalf.example.utils.MockWebServerUtil;
+
 /**
- * TODO: Add a class header comment!
+ * An example splash activity that demonstrates the simplest way to add Gandalf to a project
  */
-public class SplashActivity {
+public class SplashActivity extends GandalfActivity {
```

Comment:
I recommend we name this GandalfSplashActivity, just to distinguish it as we (and likely almost everyone else) always name our own SplashActivity the exact same thing. I know it's long but I think in this case it makes sense